README.md

```diff
@@ -123,6 +123,16 @@ to specify the database connection to use.
   end
 ```
 
+**Exclude Tables** If a database contains tables that are irrelevant to your tests, you can use the optional `exclude_tables` parameter
+to specify a set of tables to exclude from the fixpoint.
+
+```ruby
+  it 'excludes versions' do
+    restore_fixpoint :registered_user, exclude_tables: ['versions']
```

Comment:
this merge request introduces the possibility remove tables from _storing_ the fixpoint and to remove tables from _loading_ a fixpoint. I am not sure why we need both. Either I do not store the tables' data (and they subsequently not restored) or they are saved to the yaml and we ignore them from being restored. what do we want to achieve?

Reply:
Good catch, that is a mistake.  The code only allows exclude_tables for storing, not loading.